multinode_runfabtests.sh

```diff
@@ -24,3 +32,16 @@ if [ ${PROVIDER} == "efa" ]; then
     fi
 fi
 bash -c "$runfabtests_script ${FABTESTS_OPTS} ${PROVIDER} ${SERVER_IP} ${CLIENT_IP}"
+
+# dgram_pingpong test has been excluded during installation
+# (in install-fabtests.sh), because it does not work with "-E" option.
+# So here we run it separately using "-b" option
+echo "Run fi_dgram_pingpong with out-of-band synchronization"
+dgram_pingpong="${HOME}/libfabric/fabtests/install/bin/fi_dgram_pingpong"
+${dgram_pingpong} -k -p efa -b &
```

Comment:
To be consistent with runfabtests.sh should this be `ssh ${SERVER_IP} <command>` instead?

Reply:
Sure.

Reply:
Done in new revision